vendor/github.com/ceph/go-ceph/rbd/snapshot_octopus.go

```diff
@@ -1,5 +1,6 @@
-// +build octopus
+// +build rhcs_next
```

Comment:
My suggestion was to **add** this build tag, so it would look like this:
```
// +build octopus
// These snapshot APIs are not available in the RHCS build used by OCS.
// +build rhcs_next
```

Multiple `+build` tags on different lines all need to be met, see https://golang.org/cmd/go/#hdr-Build_constraints